platforms/FreeBSD/makePackage

```diff
@@ -78,6 +78,23 @@ cd "$CURRENT_DIRECTORY"
 SWIFT_VERSION="$("$SWIFT_TOOLCHAIN_HOME/bin/swift" --version | grep -Eo 'version [0-9.]+' | sed 's/version //')"
 echo "Swift version being packaged is $SWIFT_VERSION"
 
+# Get the currently installed version for all of the packages that we need.
+# This script assumes that it was run on the build machine and therefore
+# that the packages on this machine correspond to those Swift was built
+# against.
+get_package_version()
+{
+  echo "$(pkg info "$1" | grep Version | sed 's/Version.*: //')"
+}
+
+LIBUUID_PKG_VERSION=$(get_package_version libuuid)
+LIBFFI_PKG_VERSION=$(get_package_version libffi)
+LIBXML2_PKG_VERSION=$(get_package_version libxml2)
```

Comment:
This one I am weary about. Can we pin this to the exact version? I learnt that this library is not ABI stable and does not follow standard versioning practices. We are pinned to the exact version of libxml2 on Windows as there were ABI changes that break FoundationXML.